src/lwc/geBatchGiftEntryTable/geBatchGiftEntryTable.js

```diff
@@ -60,10 +64,22 @@ export default class GeBatchGiftEntryTable extends LightningElement {
     @api expectedCount;
     @track isLoaded = true;
 
+    fieldMappings;
+
+    constructor() {
+        super(DATAIMPORT_INFO.objectApiName);
+    }
+
     connectedCallback() {
-        if (this.batchId) {
-            this.loadBatch();
-        }
+        GeFormService.getFormTemplate()
+            .then( response => {
+                    this.fieldMappings = response.fieldMappingSetWrapper.fieldMappingByDevName;
+
```

Comment:
Is this new `fieldMappings` field being used in the component?  I might have missed it.  I've been trying to avoid retrieving the field mappings in the table component since they are already retrieved by the renderer (which is a sibling to this component in batch mode) and the retrieval is kind of resource expensive.  If we need the field mappings in this component do you think we should import them at the top-level and pass the data structure to child components?

Reply:
@kenlewis-sfdo good catch! We actually don't need to do this anymore. I changed the approach and forgot to remove it from the connectedCallback. Thanks!